projects/swimlane/ngx-cron/src/lib/ngx-cron.service.ts

```diff
@@ -215,35 +264,49 @@ export class NgxCronService {
     this.timezone = timezone;
   }
 
-  private validate(data: any): boolean {
-    if (data.min || data.minuteInterval > 59 || data.minuteInterval === 0) {
-      return false;
-    }
-    if (data.hour > 24) {
-      return false;
-    }
+  validateCronExpression(cron, isQuartz, configOverrides) {
```

Comment:
suggestion (non-blocking): we could just mutate `this.cronValidateConfig.override` directly then pass `this.cronValidateConfig` to `CronValidate()`?

Reply:
Done